src/data/markdown/translated-guides/en/02 Using k6/05 Options.md

```diff
@@ -983,6 +995,24 @@ $ k6 run --stage 5s:10,5m:20,10s:5 ~/script.js
 $ K6_STAGES="5s:10,5m:20,10s:5" k6 run ~/script.js
 ```
 
+```bash
+c://k6> k6 run --stage 5s:10,5m:20,10s:5 ~/script.js
+
+# or...
+
+c://k6> set "K6_STAGES=5s:10,5m:20,10s:5" && k6 run ~/script.js
+
+```
+
+```bash
+c://k6> k6 run --stage 5s:10,5m:20,10s:5 ~/script.js
+
+# or...
+
+c://k6> $env:K6_STAGES="5s:10,5m:20,10s:5" ; k6 run ~/script.js
+
+```
+
```

Comment:
I am not really of the opinion that we should have an example of doing this at all ... or have the functionality as well so ... slight :-1: and request to just remove the documentation around `K6_STAGES`.  Possibly not in this PR

Reply:
I imagine if it becomes a new PR it'll take equally long time before it's actually removed :D It's easy enough to remove. I'll leave it up to master @ppcano to decide.

Reply:
@MStoykov Please, can you open a new issue?